finality-aleph/Cargo.toml

```diff
@@ -10,6 +10,8 @@ aleph-bft = "0.18.1"
 aleph-bft-rmc = "0.4.0"
 aleph-primitives = { package = "primitives", path = "../primitives" }
 
+aleph-aggregator = { path = "../aggregator", package = "aggregator" }
+
```

Comment:
I think this line is fine, but the one two lines above should be deleted, no?

Reply:
Yeah, essentially why is `aggregator` in its own very separate line, shouldn't it be right next to the ones above?